### DISCUSSION TITLE
Official logo

### DISCUSSION THREAD

**Discussion by Tolriq** [opensubsonic/open-subsonic-api#28] [General]
In order to show support on client / server apps and documentation and increase awareness, a proper official logo would be nice.

Deluan added one as fav icon for the doc, but I'm not sure it free (and would require some cleanup) and distinctive enough.

Any thought / ideas?

**Comment by lachlan-00** [comment]
an open submarine is a boat so a little sailboat logo (possibly underwater) would be a little iteration on the old logo

**Reply by lachlan-00** [reply]
I AI generated my sub-boat and i think something like this is actually really good.

I would absolutely pay an artist to do something like this and make decent svgs sources.

<img width="1536" height="1024" alt="image" src="https://github.com/user-attachments/assets/21ebe9f8-d237-4ebc-beb9-24f68497f3d7" />

**Comment by CharlesWiltgen** [comment]
Hey @Tolriq, this is the most recent conversation I can find about an OpenSubsonic logo, which I'd like to support in a client that I'm building. Is there currently something resembling an "official" logo (ideally as SVG) that you could point me to? If there's not, are you open to help with that?
